[2014-05-07 09:31:02] log.ERROR: exception 'ErrorException' with message 'Invalid argument supplied for foreach()' in /var/www/arp_dev/app/storage/views/17ef0c71c5209e5c02622880cfcd5a37:87 Stack trace: #0 /var/www/arp_dev/app/storage/views/17ef0c71c5209e5c02622880cfcd5a37(87): Illuminate\Exception\Handler->handleError(2, 'Invalid argumen...', '/var/www/arp_de...', 87, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-05-07 11:54:52] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Database\Eloquent\Collection::$invoiceid' in /var/www/arp_dev/app/models/Mapayment.php:47 Stack trace: #0 /var/www/arp_dev/app/models/Mapayment.php(47): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 47, Array) #1 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(476): Mapayment->paymentInvoiceid() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #7 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #12 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #19 {main} [] []
[2014-05-07 11:55:34] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Database\Eloquent\Collection::$invoiceid' in /var/www/arp_dev/app/models/Mapayment.php:47 Stack trace: #0 /var/www/arp_dev/app/models/Mapayment.php(47): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 47, Array) #1 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(476): Mapayment->paymentInvoiceid() #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #7 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #12 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #19 {main} [] []
[2014-05-07 11:56:47] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Database\Eloquent\Collection::$invoiceid' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:476 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(476): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 476, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-05-07 11:57:11] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Database\Eloquent\Collection::$invoiceid' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:476 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(476): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 476, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
